iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/json/OutputDeclarationImplTest.java

```diff
@@ -208,10 +208,10 @@ void shouldFailOnMissingMandatoryAttribute() {
       "  }",
       "}");
     ParseException parseException = catchThrowableOfType(() -> parser.parse(code, null), ParseException.class);
-    assertThat(parseException).hasMessage("Missing mandatory attribute 'type' at null:3:4");
+      assertThat(parseException).hasMessage("Missing mandatory attribute 'type' at null:3:21");
```

Comment:
I tried to merge two overloads of `Arm(Json)BaseConverter#toStringLiteralOrException` into one; not sure if it had a lot of meaning to highlight the entire `TupleTree` in case of an exception instead of `TupleTree#value`

Reply:
Fine for me